samples/README.md

```diff
@@ -0,0 +1,28 @@
+# Samples, Snippets, and How-To Guides
+
+Developers like to learn by looking at code, and so the Azure SDK comes with a myriad of code samples in the form of short code snippets, sample applications, and how-to guides. This document describes where to find all these resources.
+
+## Structure of the Repository
+The Azure SDK repository is organized in the following folder structure, with the main sample locations highlighted using **bold** font.
+
+- `/samples` (this folder)
+  - `README.md` (this file)
+- `/sdk` (folder containing sources for all SDK packages)
+  - `/<service>` (e.g. storage)
+    - `/<package>` (e.g. `@azure/storage-blob`)
+      - **`README.md`** (package READMEs contain "hello world" samples)
+      - **`/samples`** (package-specific samples)
```

Comment:
This made me think that `samples` folder has some samples and then the `typescript` and `javascript` folders had more samples.

Maybe skip the text in the parenthesis here?